lib/cartopy/crs.py

```diff
@@ -1702,8 +1786,21 @@ def __init__(self, central_longitude=0, globe=None):
                           'projection may be unreliable and should be used '
                           'with caution.')
 
+        if globe is None:
+            globe = Globe(semimajor_axis=WGS84_SEMIMAJOR_AXIS, ellipse=None)
+
+        # TODO: Let the globe return the semimajor axis always.
```

Comment:
Given your knowledge of the proj4 API, is this a thing you would be willing to take a look at?
(that is: return the semimajor axis for any "globe-like" proj4 definition).

Having had a quick look at this in https://proj4.org/development/reference/datatypes.html, I don't see quite what I'm looking for though, so perhaps it doesn't exist.

Reply:
Yea, I'm not sure I see it either; it might need to be calculated manually.

Reply:
OK, hold fire for now. Perhaps we should track this in an issue.

Reply:
That'd be #330.